Documentation/Contributors/BuildGuide/README.md

```diff
@@ -44,9 +45,26 @@ Cesium ships with a simple HTTP server for testing, run `npm start` after buildi
 
 ```
 npm start
-```
+``` 
+
+Then browse to [http://localhost:8080/](http://localhost:8080/). You should see the following:
+
+* **Apps**
+   * **Cesium Viewer** : a sample application using asynchronous module definitions (AMD) to create a Cesium 3D globe viewer with some extra features
+   * **Hello World** : an example using the combined and minified Cesium library to create a 3D globe. [Tutorial here](http://cesiumjs.org/tutorials/cesium-up-and-running/)
+   * **Sandcastle** : an app for viewing and creating [code examples](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Hello%20World.html&label=Showcases), complete with a live preview
+   * **Cesium Inspector** : an example for using the Cesium Inspector widget to enable debugging features to better understand technical details of the scene
```

Comment:
Throughout, the inspector and timeline demos are just not important enough to mention.  If we cleaned things up, they probably wouldn't even be in the index.html.

So I would just say that the landing page includes links to things commonly used during development, including:
* Hello World...
* Sandcastle...
* Tests, and link to https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/TestingGuide/README.md#testing-guide
* Reference documentation built from source, and a link to https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/DocumentationGuide/README.md#documentation-guide